services/victoriametrics/prometheus.go

```diff
@@ -204,7 +204,7 @@ func AddInternalServicesToScrape(cfg *config.Config, s models.MetricsResolutions
 		scrapeConfigForPMMManaged(s.MR),
 		scrapeConfigForQANAPI2(s.MR),
 	)
-	// TODO Refactor to remove boolean positional parameter when Prometheus or PERCONA_TEST_DBAAS is removed
+	// TODO Refactor to remove boolean positional parameter when Prometheus is removed
 	if dbaas {
```

Comment:
Do we regenerate scrape config on enable/disable DBaaS?

Reply:
Yes, we do. Does it affect something else than dbaas-controller? Only difference is we scrape dbaas-controller only if it's running.

Reply:
Yes, we do as part of server.UpdateConfigurations method.